app/views/mentor/knowledges/new.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_for ['mentor', @knowledge] do |f|
+  = f.text_field :title, placeholder: "title"
+  = f.text_area :content, cols: "100", row: "100"
+  = f.submit '送信'
+
+
+%div{id: "preview"}
```

Comment:
__[HAML-Lint]__

Hash attribute should start with one space after the opening brace


<!-- Comment by [SideCI](https://sideci.com) -->